.changeset/mighty-donuts-behave.md

```diff
@@ -0,0 +1,33 @@
+---
+"@effect/platform": minor
+"effect": minor
+"@effect/schema": minor
+"@effect/cli": minor
+"@effect/rpc": minor
+---
+
+With this change we remove the `Data.Data` type and we make `Equal.Equal` & `Hash.Hash` an implicit traits.
+
+The main reason is that `Data.Data<A>` was structurally equivalent to `A & Equal.Equal` but extending `Equal.Equal` doesn't mean that the equality is implemented by-value, so the type was simply adding noice without gaining any level of safety.
+
+The module `Data` remains unchanged at the value level, all the functions priorly available are supposed to work in exactly the same manner.
```

Comment:
```suggestion
The module `Data` remains unchanged at the value level, all the functions previously available are supposed to work in exactly the same manner.
```

Reply:
Uuuh, Mr. Fancy Pants ... "priorly"  :-P

Reply:
let the man priorize as he pleases

Reply:
priorly is proper english but whatever

Reply:
It is. But not for numbnuts like me. I want my sentences simple and easy to grok.